src/dialogs/notifications/notification-item-template.ts

```diff
@@ -25,14 +25,11 @@ export class HuiNotificationItemTemplate extends LitElement {
     }
 
     ha-card .header {
-      font-family: var(--paper-font-headline_-_font-family);
-      -webkit-font-smoothing: var(
-        --paper-font-headline_-_-webkit-font-smoothing
-      );
-      font-size: var(--paper-font-headline_-_font-size);
-      font-weight: var(--paper-font-headline_-_font-weight);
-      letter-spacing: var(--paper-font-headline_-_letter-spacing);
-      line-height: var(--paper-font-headline_-_line-height);
+      font-family: var(--ha-font-family-body);
```

Comment:
I would really like to see a header font variable, that could be by default set to the body font family...

Reply:
Sure let's add it :+1: